packages/manager/modules/pci/src/projects/project/storages/databases/database/pools/add-edit/add-edit.routing.js

```diff
@@ -0,0 +1,43 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.storages.databases.dashboard.pools.add', {
```

Comment:
**question:** Why using single AngularJS module/routing/component to hande two separate behavior (creation and edition)? Assuming that edition needs to have additional changes, this will leas us to non-necessary complexity.

Reply:
@antleblanc Since the page is similar for both addition and updation of Pool, an internal boolean flag is used and handled both the case in the single component.

Reply:
**issue(blocking):** Both actions must be split into its own module. We took this approach for consistency and maintainability.